src/lib/sql/schemas.sql

```diff
@@ -1,12 +1,15 @@
+-- Adapted from infromation_schema.schemata
+
 SELECT
-  nsp.oid AS id,
-  catalog_name,
-  schema_name AS name,
-  schema_owner AS owner,
-  default_character_set_catalog,
-  default_character_set_schema,
-  default_character_set_name,
-  sql_path
```

Comment:
Unsupported in Postgres ([ref](https://www.postgresql.org/docs/13/infoschema-schemata.html))